panels/user-accounts/cc-app-permissions.c

```diff
@@ -433,7 +434,18 @@ update_allow_app_installation (CcAppPermissions *self)
 static void
 setup_parental_control_settings (CcAppPermissions *self)
 {
+  g_autoptr(GDBusConnection) system_bus = NULL;
   gboolean is_authorized, user_is_administrator;
+  g_autoptr(GError) error = NULL;
+
+  system_bus = g_bus_get_sync (G_BUS_TYPE_SYSTEM, self->cancellable, &error);
+  if (system_bus == NULL)
+    {
+      g_warning ("Error getting system bus for setting up parental controls: %s", error->message);
```

Comment:
Rather than introducing the possibility of emitting a warning here, would it be possible to set up a single `GDBusConnection` instance for the system bus when the panel is first instantiated, and just use it here?

Reply:
Yes, we now instantiate `MctManager` in `cc_app_permissions_init`. This entire hunk is entirely moved to `_init ()` in the latest iteration of the patch.